pgml-extension/src/orm/model.rs

```diff
@@ -158,9 +158,21 @@ impl Model {
     }
 
     #[allow(clippy::too_many_arguments)]
-    pub fn tune(project: &Project, snapshot: &mut Snapshot, hyperparams: &JsonB) -> Model {
+    pub fn finetune(project: &Project, snapshot: &mut Snapshot, hyperparams: &JsonB) -> Model {
```

Comment:
We should update our docs as well. Currently, we refer to it as `pgml.tune()`.

Reply:
User facing function is still `pgml.tune`.